config/config/sc-config.yaml

```
@@ -718,6 +718,13 @@ opensearch:
   defaultTemplates: true
   additionalTemplates: {}
 
+  plugins:
+    # in an air-gapped environment where the nodes are not connected to the Internet , the following
+    # variable can be used to override where the objectStorage plugin should be downloaded from.
+    objectStorageUrlOverride: # e.g. server.local:8080/repository-s3-2.8.0.zip
+
+    additionalPlugins: []
+
```

```suggestion
  plugins:
    # in an air-gapped environment where the nodes are not connected to the Internet, 
    # this will prevent external/default object storage plugins from being installed
    installExternalObjectStoragePlugin: false
    # or installDefaultObjectStoragePlugin: false
    # or installObjectStoragePlugin: false

    # in an air-gapped environment this can be used to install plugins from known sources
    additionalPlugins: []
      # - server.local:8080/repository-s3-2.8.0.zip
```

Can we rework this a bit and then just reuse the `additionalPlugins`?